src/composables/multisigAccounts.ts

```diff
@@ -207,79 +292,12 @@ export function useMultisigAccounts({
-    const result: IMultisigAccount[] = (await Promise.all(
-      rawMultisigData
-        .filter(({ version }) => version === MULTISIG_SUPPORTED_CONTRACT_VERSION)
-        .map(async ({
-          contractId,
-          gaAccountId,
-          ...otherMultisigData
-        }): Promise<IMultisigAccount> => {
-          try {
-            const contractInstance = await dryAeSdk.initializeContract({
-              aci: SimpleGAMultiSigAci,
-              address: contractId,
-            });
-
-            const currentAccount = multisigAccounts.value
-              .find((account) => account.contractId === contractId);
-
-            const [
-              nonce,
-              signers,
-              consensusResult,
-              balance,
-            ] = (await Promise.all([
-              (
-                (isAdditionalInfoNeeded.value && gaAccountId === activeMultisigAccountId.value)
-                || currentAccount?.nonce == null
-              )
-                ? contractInstance.get_nonce()
-                : { decodedResult: currentAccount.nonce },
-              currentAccount?.signers
-                ? { decodedResult: currentAccount.signers }
-                : contractInstance.get_signers(),
-              contractInstance.get_consensus_info(),
-              gaAccountId ? dryAeSdk.getBalance(gaAccountId as Encoded.AccountAddress) : 0,
-            ]));
-
-            const decodedConsensus = consensusResult.decodedResult;
-            const txHash = decodedConsensus.tx_hash as Uint8Array;
-            const consensus: IMultisigConsensus = camelCaseKeysDeep(decodedConsensus);
-
-            consensus.expirationHeight = Number(consensus.expirationHeight);
-            consensus.confirmationsRequired = Number(consensus.confirmationsRequired);
-
-            const hasPendingTransaction = !!txHash && !consensus.expired;
-
-            return {
-              ...consensus,
-              ...otherMultisigData,
-              contractId,
-              gaAccountId,
-              nonce: Number(nonce.decodedResult),
-              signers: signers.decodedResult,
-              balance: toShiftedBigNumber(balance, -AE_COIN_PRECISION),
-              hasPendingTransaction,
-              txHash: txHash ? Buffer.from(txHash).toString('hex') : undefined,
-            };
-          } catch (error) {
-            /**
-             * Node might throw nonce mismatch error, skip the current account update
-             * return the existing data and account details will be updated in the next poll.
-             */
-            if (!(error instanceof DryRunError)) {
-              handleUnknownError(error);
-            }
-            return multisigAccounts.value.find(
-              (account) => account.contractId === contractId,
-            ) as IMultisigAccount;
-          }
-        }),
-    ))
+    const splittedMultisig = chunk(rawMultisigData
+      .filter(({ version }) => version === MULTISIG_SUPPORTED_CONTRACT_VERSION), 5);
```

Comment:
I think the logic of splitting the accounts in chunks should be moved to `getAllMultisigAccountsInfo`.